Peer.Domain/Validators.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Peer.Domain
+{
+    [ExcludeFromCodeCoverage]
+    public class Validators
+    {
+        public const string NotLessThanOrEqualToZero = "Cannot be less than or equal to zero";
+        public const string NotNullEmptyOrWhitespace = "Cannot be null, empty or only whitespace";
+        public const string NotGuidEmpty = "Cannot be Guid.Empty";
+        public const string NotEmpty = "Cannot be empty";
+        public const string UndefinedEnum = "Enum value cannot be undefined";
+
+        public static void ArgNotLessThanOrEqualToZero(int value, string name)
+        {
+            if (value <= 0)
+                throw new ArgumentException(NotLessThanOrEqualToZero, name);
+        }
+
+        public static void ArgIsNotNullEmptyOrWhitespace(string value, string name)
+        {
+            if (value == null)
+                throw new ArgumentNullException(name);
+
+            if (string.IsNullOrWhiteSpace(value))
```

Comment:
This will never throw for `null` because of the check above. The message is still true but it might be more fitting to just call it an empty/whitespace check.